docs/core/tools/dotnet.md

```diff
@@ -96,31 +103,27 @@ The following options are for `dotnet` with a command. For example, `dotnet buil
 
 - **`-?|-h|--help`**
 
-  Prints out documentation for a given command, such as `dotnet build --help`.
+  Prints out documentation for a given command. For example, `dotnet build --help` displays help for the `build` command.
 
 - **`command options`**
 
   Each command defines options specific to that command. See specific command page for a list of available options.
 
-### Runtime options
+### Options for running an application
+
+The following options are available when `dotnet` runs an application with or without the `exec` command. For example, `dotnet exec --roll-forward Major myapp.dll` or `dotnet --roll-forward Major myapp.dll`.
```

Comment:
I'd remove `exec` here. I've never been entirely clear on its primary use case, but I don't think its the one we want to offer as the go-to command. The other one is good.

@vitek-karas